core/templates/pages/landing-pages/topic-landing-page/topic-landing-page.constants.ts

```diff
@@ -23,72 +23,44 @@ export class TopicLandingPageConstants {
   public static TOPIC_LANDING_PAGE_DATA = {
     math: {
       fractions: {
-        topic_title: 'Fractions',
-        topic_tagline: 'Add, Subtract, Multiply and Divide',
-        collection_id: '4UgTQUc1tala',
-        page_data: {
-          image_1: {
-            file_name: 'matthew_paper.png',
-            alt: 'Matthew showing parts of fractions written on a card.'
-          },
-          image_2: {
-            file_name: 'matthew_fractions.png',
-            alt: 'Matthew solving problems on Oppia.'
-          },
-          video: 'fractions_video.mp4',
-          lessons: [
-            'What is a Fraction?',
-            'Comparing Fractions',
-            'The Meaning of "Equal Parts"',
-            'Adding & Subtracting Fractions'
-          ]
-        }
+        topicTitle: 'Fractions',
+        topicTagline: 'Add, Subtract, Multiply and Divide',
+        collectionId: '4UgTQUc1tala'
       },
       'negative-numbers': {
-        topic_title: 'Negative Numbers',
-        topic_tagline: '',
-        collection_id: 'GdYIgsfRZwG7',
-        page_data: {
-          image_1: {
-            file_name: 'negative_1.png',
-            alt: 'A boy showing 3 + -24 written on a slate.'
-          },
-          image_2: {
-            file_name: 'negative_2.png',
-            alt: 'A boy smiling and solving negative-number problems on Oppia.'
-          },
-          video: 'negative-numbers_video.mp4',
-          lessons: [
-            'The Number Line',
-            'What is a Negative Number?',
-            'Adding & Subtracting Negative Numbers',
-            'Multiplying & Dividing Negative Numbers'
-          ]
-        }
+        topicTitle: 'Negative Numbers',
+        topicTagline: '',
```

Comment:
Add, Subtract, Multiply and Divide

Reply:
It was left empty earlier, thanks for the suggestion, I'll add this!